server/channels/app/platform/options.go

```diff
@@ -51,6 +51,9 @@ func StoreOverrideWithCache(override store.Store) Option {
 			if err != nil {
 				return nil, err
 			}
+			// Clearing all the caches because the in-mem data
+			// is persisted in case of Redis.
+			lcl.Invalidate()
```

Comment:
I assume this service option is used to support test scenarios only? If so, could we make it more explicit? (e.g. a comment or renaming it)

Reply:
Sure